plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -174,6 +174,20 @@ fu_plugin_thunderbolt_find_nvmem (GUdevDevice  *udevice,
 	return NULL;
 }
 
+static gchar *
+fu_plugin_thunderbolt_parse_version (const gchar *version_raw)
+{
+	g_auto(GStrv) split = NULL;
+	if (version_raw == NULL)
+		return NULL;
+	split = g_strsplit (version_raw, ".", -1);
+	if (g_strv_length (split) != 2)
+		return NULL;
+	return g_strdup_printf ("%02x.%02x",
+				(guint) g_ascii_strtoull (split[0], NULL, 16),
+				(guint) g_ascii_strtoull (split[1], NULL, 16));
+}
+
 static const gchar *
```

Comment:
this can't be const if it's returning an allocated string

Reply:
@hughsie
"Can't" technically or by convention?